SpliceRegion.pm

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 
  mv SpliceRegion.pm ~/.vep/Plugins
  ./vep -i variations.vcf --plugin SpliceRegion
+ 
+ To only show the additional consequence extended_intronic_splice_region_variant, use: 
```

Comment:
Would it not be better to add the new annotations to the existing ones? Or was the request for extended annotations only?

Reply:
I had a quick chat with Sarah about this - we didn't want to disrupt the previous functionality as there are a couple of users using this plugin in their pipelines

Reply:
You don't change the previous behaviour. But if you add extended you only get the newly added annotations. What if a user wants all 5 annotations splice_* and extended_*?